pkg/scm/gitlab/client_actioner_assign.go

```diff
@@ -2,15 +2,18 @@ package gitlab
 
 import (
 	"context"
+	"fmt"
 	"log/slog"
+	"strings"
 
+	"github.com/datolabs-io/go-backstage/v3"
 	"github.com/jippi/scm-engine/pkg/scm"
 	"github.com/jippi/scm-engine/pkg/state"
 	slogctx "github.com/veqryn/slog-context"
 )
 
 func (c *Client) AssignReviewers(ctx context.Context, evalContext scm.EvalContext, update *scm.UpdateMergeRequestOptions, step scm.ActionStep) error {
-	source, err := step.OptionalStringEnum("source", "codeowners", "codeowners")
+	source, err := step.OptionalStringEnum("source", "codeowners", "codeowners", "backstage")
```

Comment:
[optional; question] Why is `codeowners` in here twice? I know you didn't add it, but it caught my eye.

Reply:
Optional string enum accepts a name, a default value, and a string... of possible values, hence why its added there twice. but honestly we might as well make it required since we are introducing multiple sources of reviewers